libs/shared/ui/src/lib/components/board/board.tsx

```diff
@@ -0,0 +1,255 @@
+
+        copy.splice(insertAtIndex, 0, cardToTransfer)
+      }
+
+      setData({
+        sourceColumnId,
+        sourceCardId: cardId,
+        targetColumnId: columnId,
+        newData: copy,
+      })
+    }
+  }
+
+  const handleDragOver = (e: DragEvent) => {
+    e.preventDefault()
+
+    highlightIndicator(e)
+
+    setActive(true)
+  }
+
+  const clearHighlights = (els?: HTMLElement[]) => {
+    const indicators = els || getIndicators()
+
+    indicators.forEach((i) => {
+      i.style.opacity = '0'
+    })
+  }
+
+  const highlightIndicator = (e: DragEvent) => {
+    const indicators = getIndicators()
+
+    clearHighlights(indicators)
+
+    if (shouldHighlightIndicator) {
+      const el = getNearestIndicator(e, indicators)
+
+      el.element.style.opacity = '1'
+    }
+  }
+
+  const getNearestIndicator = (e: DragEvent, indicators: HTMLElement[]) => {
+    const DISTANCE_OFFSET = 50
+
+    const el = indicators.reduce(
+      (closest, child) => {
+        const box = child.getBoundingClientRect()
+
+        const offset = e.clientY - (box.top + DISTANCE_OFFSET)
+
+        if (offset < 0 && offset > closest.offset) {
+          return { offset: offset, element: child }
+        } else {
+          return closest
+        }
+      },
+
+      {
+        offset: Number.NEGATIVE_INFINITY,
+
+        element: indicators[indicators.length - 1],
+      }
+    )
+
+    return el
+  }
+
+  const getIndicators = (): HTMLElement[] => {
+    return Array.from(document.querySelectorAll(`[data-board="${boardId}"][data-column="${columnId}"]`))
+  }
+
+  const handleDragLeave = () => {
+    clearHighlights()
+
+    setActive(false)
+  }
+
+  return (
+    <motion.div layout="position" layoutId={columnId} className="flex w-60 shrink-0 flex-col rounded">
+      <div className="flex h-11 items-center justify-between rounded-t border border-neutral-250 bg-neutral-100 px-3 py-2">
+        {heading}
+      </div>
+
+      <div
+        onDrop={(e) => handleDragEnd(e)}
+        onDragOver={handleDragOver}
+        onDragLeave={handleDragLeave}
+        className={`w-full rounded-b border border-t-0 border-neutral-250 px-1 transition-colors ${active ? 'bg-green-100' : 'bg-neutral-200'}`}
+      >
+        {children}
+        <BoardDropIndicator columnId={columnId} />
+      </div>
+    </motion.div>
+  )
+}
+
+interface BoardCardProps extends Card, PropsWithChildren {}
+
+const BoardCard = ({ children, cardId, columnId }: BoardCardProps) => {
+  const handleDragStart = (e: DragEvent) => {
+    e.dataTransfer?.setData('cardId', cardId)
+    e.dataTransfer?.setData('columnId', columnId)
+  }
+
+  return (
+    <>
+      <BoardDropIndicator beforeId={cardId} columnId={columnId} />
+      <motion.div
+        layout
+        layoutId={cardId}
+        draggable="true"
+        onDragStart={(e) => handleDragStart(e as unknown as DragEvent)}
+        className="cursor-grab rounded border border-neutral-200 bg-neutral-50 px-2 py-3 active:cursor-grabbing"
```

Comment:
`active:` seems to not works, do you know if it's possible to have an `active:outline-green-500`, to keep the previous behavior?

Reply:
`active:` is working but you need to class `active:outline` & `active:outline-green-500`
https://github.com/Qovery/console/pull/1692/commits/a872c888e8e6e20242d6311d0b33cd472dedf9ee